gensim/topic_coherence/probability_estimation.py

```diff
@@ -70,7 +70,7 @@ def unique_ids_from_segments(segmented_topics):
     for s_i in segmented_topics:
         for word_id in itertools.chain.from_iterable(s_i):
             if hasattr(word_id, '__iter__'):
```

Comment:
Yes, they will always be integers; I have updated the docstrings to better reflect this. The segmented topics will always come from the `segmentation module`, and the setter in the `CoherenceModel` ensures the segmentation module receives lists of integer arrays to be segmented.